packages/cadl-ranch-specs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure-tools/cadl-ranch-specs",
-  "version": "0.28.7",
+  "version": "1.0.0",
```

Comment:
Please use a manual PR. This is not correct.

Reply:
resolved by edit the versions in github. checked the history, and it should work without breaking the bots for further bump.